src/utils/componentHelpers.ts

```diff
@@ -78,6 +78,18 @@ export function getUserVaultBoost(vault: VaultDTO, boost: number, apr = false):
     .reduce((total, apr) => total + apr, 0);
 }
 
+export function getProjectedVaultBoost(vault: VaultDTO, boost: number, apr = false): number {
+  const maxBoost = calculateUserBoost(MAX_BOOST_RANK.stakeRatioBoundary);
+  return (apr ? vault.sources : vault.sourcesApy)
+    .map((source) => {
+      if (source.name === 'Badger Rewards' || source.name === 'Boosted Badger Rewards') {
```

Comment:
I think we wanted to use a .includes("Badger Rewards"). im not positive on all the data values here so best to go with Jintao's suggestion